src/ConductorSharp.Toolkit/Service/ScaffoldingService.cs

```diff
@@ -0,0 +1,286 @@
+
+            Directory.CreateDirectory(dir);
+
+            File.WriteAllText(dir + Path.DirectorySeparatorChar + "Tasks.scaff.cs", taskTemplate);
+
+            var workflowTemplate = EmbeddedFileHelper.GetLinesFromEmbeddedFile("~/Templates/WorkflowCollectionTemplate.default");
+            var workflowCollection = workflowCollectionBuilder.ToString();
+            workflowTemplate = workflowTemplate.Replace("{{namespace}}", config.BaseNamespace + ".Workflows");
+            workflowTemplate = workflowTemplate.Replace("{{workflowCollection}}", workflowCollection);
+            var workflowDir = config.Destination + Path.DirectorySeparatorChar + "Workflows";
+
+            Directory.CreateDirectory(workflowDir);
+
+            File.WriteAllText(workflowDir + Path.DirectorySeparatorChar + "Workflows.scaff.cs", workflowTemplate);
+        }
+
+        public async Task<string> CreateWorkflowClass(WorkflowDefinition workflowDefinition)
+        {
+            var lines = EmbeddedFileHelper.GetLinesFromEmbeddedFile("~/Templates/WorkflowTemplate.default");
+
+            string name = SnakeCaseUtil.ToPascalCase($"{workflowDefinition.Name}_V{workflowDefinition.Version}").Trim();
+            string note = null;
+            if (LangUtils.MakeValidMemberName(name, "A", out name))
+            {
+                note = "The autogenerated name has been changed because it is invalid C# member name";
+            }
+            //name = char.ToUpperInvariant(name[0]) + name[1..];
+
+            string workflowDescription = workflowDefinition.Description;
+            string[] labels = null;
+
+            try
+            {
+                var descriptionObject = JObject.Parse(workflowDescription);
+                workflowDescription = descriptionObject.SelectToken("description")?.Value<string>();
+                var labelsObject = descriptionObject.SelectToken("labels");
+
+                if (labelsObject is JArray labelsArray)
+                {
+                    labels = labelsArray.ToObject<string[]>();
+                }
+            }
+            catch (Exception) { }
+
+            var inputLinesBuilder = new StringBuilder();
+            try
+            {
+                var inputParameters = JObject.Parse(workflowDefinition.InputParametersJSON[0]);
+
+                foreach (var param in inputParameters)
+                {
+                    var value = param.Value.SelectToken("value")?.ToString(Newtonsoft.Json.Formatting.None);
+                    var type = param.Value.SelectToken("type")?.Value<string>();
+                    var description = param.Value.SelectToken("description")?.Value<string>();
+
+                    if (type == null)
+                        type = "string";
+
+                    inputLinesBuilder.AppendLine($"        /// <originalName>");
+                    inputLinesBuilder.AppendLine($"        /// {param.Key}");
+                    inputLinesBuilder.AppendLine($"        /// </originalName>");

```

Comment:
We can extract this to a method, something like 
`AppendXmlComment(string tag, string value)`